.github/workflows/test-package.yaml

```diff
@@ -9,6 +9,17 @@ jobs:
     steps:
       - name: checkout
         uses: actions/checkout@v4
+      - name: check version
+        id: vbump
+        uses: rgryta/Check-Bump@main
+        with:
+          method: 'toml'
+          prefix: 'v'
```

Comment:
Is this prefix enough? The field is named "version" in the pyproject.toml file.

Reply:
yea, if you look at the last 2 workflow runs it can tell that version was bumped after the last commit

Reply:
also that prefix is for the output string for the action, `vbump.outputs.version`, it's not used when analyzing the pyproject.toml file.

Reply:
Ok, thank you!